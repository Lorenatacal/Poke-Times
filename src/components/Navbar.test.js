import React, { Component } from 'react';
import NavBar from './NavBar';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import { BrowserRouter as Router } from 'react-router-dom';


Enzyme.configure({ adapter: new Adapter() });

test('NavBar should render correctly', () => {
    const wrapper = Enzyme.mount(
        <Router>
            <NavBar />
        </Router>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
})