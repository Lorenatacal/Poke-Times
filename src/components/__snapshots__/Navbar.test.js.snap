// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render correctly 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <withRouter(Navbar)>
      <Route>
        <Navbar
          history={
            Object {
              "action": "POP",
              "block": [Function],
              "createHref": [Function],
              "go": [Function],
              "goBack": [Function],
              "goForward": [Function],
              "length": 1,
              "listen": [Function],
              "location": Object {
                "hash": "",
                "pathname": "/",
                "search": "",
                "state": undefined,
              },
              "push": [Function],
              "replace": [Function],
            }
          }
          location={
            Object {
              "hash": "",
              "pathname": "/",
              "search": "",
              "state": undefined,
            }
          }
          match={
            Object {
              "isExact": true,
              "params": Object {},
              "path": "/",
              "url": "/",
            }
          }
        >
          <nav
            className="nav wrapper red darken-3"
          >
            <div
              className="container"
            >
              <ul
                className="right"
              >
                <li>
                  <Link
                    replace={false}
                    to="/"
                  >
                    <a
                      href="/"
                      onClick={[Function]}
                    >
                      Home
                    </a>
                  </Link>
                </li>
                <li>
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    to="/about"
                  >
                    <Route
                      path="\\\\/about"
                    >
                      <Link
                        aria-current={null}
                        replace={false}
                        to="/about"
                      >
                        <a
                          aria-current={null}
                          href="/about"
                          onClick={[Function]}
                        >
                          About
                        </a>
                      </Link>
                    </Route>
                  </NavLink>
                </li>
                <li>
                  <NavLink
                    activeClassName="active"
                    aria-current="page"
                    to="/contact"
                  >
                    <Route
                      path="\\\\/contact"
                    >
                      <Link
                        aria-current={null}
                        replace={false}
                        to="/contact"
                      >
                        <a
                          aria-current={null}
                          href="/contact"
                          onClick={[Function]}
                        >
                          Contact
                        </a>
                      </Link>
                    </Route>
                  </NavLink>
                </li>
              </ul>
            </div>
          </nav>
        </Navbar>
      </Route>
    </withRouter(Navbar)>
  </Router>
</BrowserRouter>
`;
